modules/ROOT/pages/brand-intro.adoc

```diff
@@ -0,0 +1,14 @@
+= Brand and Customize Your API Community
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+API Community Manager allows you to customize and adjust the branding, look and feel, and site structure of your API community to the finest level of detail, using a simple point-and-click approach.
+
+Advanced HTML and CSS editing is also available for very precise web designs.
+
```

Comment:
General comment: Use superlatives sparingly, so that your audience approaches your documentation with an appropriate level of gravitas. Reconsider "finest" and omit "very" (precise is precise).  :-)

Reply:
Much better! Fixed.